ci/ray_ci/doc_builder_container.py

```diff
@@ -0,0 +1,15 @@
+from ci.ray_ci.container import Container
+
+
+class DocBuilderContainer(Container):
+    def __init__(self) -> None:
+        super().__init__("docbuild")
+        # self.install_ray()
+
+    def run(self) -> None:
+        self.run_script(
+            [
+                "cd doc",
+                "FAST=True make html",
```

Comment:
ooc, what does `FAST=True` mean?

Reply:
it will omit some source files from compilation https://github.com/ray-project/ray/blob/master/doc/source/conf.py#L78